docs/examples/forecasting/extended_tutorial.md

```diff
@@ -0,0 +1,1746 @@
+
+
+```python
+[k for k in next(iter(train_ds)).keys()]
+```
+
+Now, we can move on and see how the test dataset is split. As we saw, the transformation splits the windows to past and future. However, during inference (`is_train=False` in the transformation), the splitter always cuts the last window (of length `context_length`) of the dataset so it can be used to predict the subsequent unknown values of length `prediction_length`. 
+
+So, how is the test dataset split in past and futere since we do not know the future target? And what about the time dependent features?
+
+
+```python
+test_tf = transformation(iter(test_ds), is_train=False)
+```
+
+
+```python
+test_tf_entry = next(iter(test_tf))
+[k for k in test_tf_entry.keys()]
+```
+
+
+```python
+print(f"past target shape: {test_tf_entry['past_target'].shape}")
+print(f"future target shape: {test_tf_entry['future_target'].shape}")
+print(f"past observed values shape: {test_tf_entry['past_observed_values'].shape}")
+print(f"future observed values shape: {test_tf_entry['future_observed_values'].shape}")
+print(f"past age feature shape: {test_tf_entry['past_feat_dynamic_age'].shape}")
+print(f"future age feature shape: {test_tf_entry['future_feat_dynamic_age'].shape}")
+print(test_tf_entry['feat_static_cat'])
+```
+
+The future target is empty but not the features - we always assume that we know the future features!
+
+All the things we did manually here are done by an internal block called `DataLoader`. It gets as an input the raw dataset (in appropriate format) and the transformation object and it outputs the transformed iterable dataset batch by batch. The only thing that we need to worry about is setting the transformation fields correctly! 
+
+
+```python
+
+```
+
+# 3. Training an existing model
+
+As we already mentioned, GluonTS comes with a number of pre-built models that can be used directly with minor hyperparameter configurations. For starters we will use one of these predefined models to go through the whole pipeline of training a model, predicting, and evaluating the results. 
+
+GluonTS gives focus (but is not restricted) to probabilistic forecasting, i.e., forecasting the future distribution of the values and not the future values themselves (point estimates) of a time series. Having estimated the future distribution of each time step in the forecasting horizon, we can draw a sample from the distribution at each time step and thus create a "sample path" that can be seen as a possible realization of the future. In practice we draw multiple samples and create multiple sample paths which can be used for visualization, evaluation of the model, to derive statistics, etc.
+
+In this example we will use a simple pre-built feedforward neural network estimator that takes as input a window of length `context_length` and predicts the distribution of the values of the subsequent future window of length `prediction_length`. 
```

Comment:
comment here by Alex Cooper was to reference back the estimator/predictor terminology to scikit-learn/spark ML as this may be a bit foreign for the econometricians and statisticians (where a predictor is a feature or a co-variate for example).